services/director-v2/src/simcore_service_director_v2/modules/db/repositories/comp_tasks.py

```diff
@@ -102,7 +102,7 @@ async def _generate_tasks_list_from_project(
 
         task_state = node.state.current_status
         if node_id in published_nodes and node_class == NodeClass.COMPUTATIONAL:
-            task_state = RunningState.PUBLISHED
+            task_state = RunningState.PENDING
```

Comment:
Reading line 104, i would assume that the state is PUBLISHED ... why was it changed?

NOTE: at some point it would be good to document in ``packages/models-library/src/models_library/projects_state.py`` each state.

Reply:
because:
- before the director would PUBLISH the workflow, then the sidecar would pick it up and create the actual tasks and set to PENDING
- after that change, the director directly creates the tasks and set them to PENDING